.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,6 +1,6 @@
 name: 🐞 Bug Report
 description: Report an issue
-labels: ["awaiting-triage", "bug 🐞"]
+labels: ["bug 🐞"]
```

Comment:
> We're switching to use a Project status field for this purpose

I had to dig a little to find this, so posting for others to see too: https://github.com/orgs/openxla/projects/13/views/1
![image](https://user-images.githubusercontent.com/4010439/230152150-488a6dde-faed-4717-b699-064a70e6adcf.png)